README.md

```diff
@@ -1,2 +1,13 @@
 # Toxicity-Classifier
  Classifier of toxic language
+
+## Development
+
+### Setup
+These were confirmed on macOS Catalina with both python (2.2.7) and python3 (3.7.5) binaries installed. You may need to use just `python` instead of `python3` and `pip` instead of `pip3` depending on your environment:
+- `virtualenv --python=/usr/bin/python3 .`
```

Comment:
Maybe after this also the command for activating the virtual environment, to avoid confusion:

`source bin/activate`

Once the environment is activated, `python` and `pip` point to the specified Python version, so `python3` etc. shouldn't be necessary.

I think.